kubernetes.yml

```diff
@@ -59,6 +59,26 @@ spec:
             secretKeyRef:
               name: aprb-credentials
               key: gravity_api_token
+        - name: RABBITMQ_HOST
+          valueFrom:
+              secretKeyRef:
+                name: aprb-credentials
+                key: rabbitmq_host
+        - name: RABBITMQ_USER
+          valueFrom:
+              secretKeyRef:
+                name: aprb-credentials
+                key: rabbitmq_user
+        - name: RABBITMQ_PASSWORD
+          valueFrom:
+              secretKeyRef:
+                name: aprb-credentials
+                key: rabbitmq_password
+        - name: RABBITMQ_VHOST
```

Comment:
Actually thinking more about this, I think it's good to still have it as an env variable in case we wanted to setup staging, dev deployments.